site/en/docs/devtools/elements/container-queries/index.md

```diff
@@ -13,7 +13,7 @@ tags:
 
 This guide shows you how to inspect and debug CSS container queries in the **Elements** panel in Chrome DevTools.
 
-[CSS container queries](https://developer.mozilla.org/docs/Web/CSS/CSS_Container_Queries) allow you to manipulate the element's styles based on its parent container properties. This capability shifts the concept of [responsive web design][2] from page-based to [container-based](https://web.dev/new-responsive/).
+[CSS container queries][3] allow you to manipulate the element's styles based on its parent container properties. This capability shifts the concept of [responsive web design][2] from page-based to [container-based](https://web.dev/new-responsive/).
 
 {% Aside %}
 CSS container queries is an experimental feature. To enable its support in DevTools, under [`chrome://flags`](chrome://flags), set the `Enable CSS Container Queries` experiment to `Enabled`.
```

Comment:
[`chrome://flags`](chrome://flags), set the `Enable CSS Container Queries` experiment to `Enabled`.
Write Previewv